frontend/test/playwright/playwright.config.ts

```diff
@@ -21,6 +21,8 @@ const pwCommand = process.env.FASTSTART !== "false" ? "dev" : "prod:playwright"
 
 const config: PlaywrightTestConfig = {
   forbidOnly: !!process.env.CI,
+  snapshotPathTemplate:
```

Comment:
By default, this is `'{snapshotDir}/{testFileDir}/{testFileName}-snapshots/{arg}{-projectName}{-snapshotSuffix}{ext}'`

`snapshotSuffix` is the platform (`linux`), `project` is a logical group of tests running in the same browser (context). Since we only have one project, we don't need this.

https://github.com/microsoft/playwright/blob/a4bd551597f0161a853f7bdb725e738544db2130/packages/playwright/src/common/config.ts#L166

Reply:
Removed this in [6395b2c](https://github.com/WordPress/openverse/pull/4914/commits/6395b2c7c73d697b8e1e7c8138e7e6d9c81df788)